src/main/java/io/candydoc/ddd/core_concept/CoreConceptInteractionStrategy.java

```diff
@@ -71,6 +72,14 @@ public Event domainEvent(DomainEvent domainEvent) {
                             .build();
                       }
 
+                      @Override
```

Comment:
I think it is an other issue I created: tomorrow, the only interactions a core concept could have would be the same as the value object (and maybe the aggregate)